pages/about.tsx

```diff
@@ -1,5 +1,9 @@
 import React from "react";
 
-export default function about() {
-  return <div>about</div>;
+export default function About() {
+  return (
+    <div>
+      <h1>Welcome to Wordify! This is the about page.</h1>
```

Comment:
You should maybe add a description / overview of the project, including that this is our CS32 term project. I believe this was in the handout as one of our requirements!